kimchi/src/prover.rs

```diff
@@ -1328,6 +1366,36 @@ where
                     runtime_table_comm.blinders.clone(),
                 ));
             }
+
+            //~~ * the lookup selectors
+
+            if let Some(runtime_lookup_table_selector) = lcs.runtime_selector.as_ref() {
+                polynomials.push((
+                    evaluations_form(runtime_lookup_table_selector),
+                    None,
+                    non_hiding(1),
```

Comment:
The `non_hiding` mode is meant for circuit-related polynomials, as opposed to prover-specific's, right?